stylesheet/imports.css

```diff
@@ -16,4 +16,4 @@
  * CSS imports for FirebaseUI Javascript widget.
  */
 
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap);
+@import url(https://fonts.googleapis.com/css?family=Roboto:400,500,700);
```

Comment:
`&display=swap` was added by an external contributor. let's revert that change.
https://github.com/firebase/firebaseui-web/commit/540ddc0927920b0380fc9ba6d3f08d1377985fcc#diff-02bb13a64e6befcfef900fdb5e35cb6b8db523c58bc11755c548bc4fbcdce2a8